pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2260,20 +2294,32 @@ func (s *AWSCloud) GetLoadBalancer(service *api.Service) (*api.LoadBalancerStatu
 		return nil, false, nil
 	}
 
-	status := toStatus(lb)
+	status := s.toStatus(lb, service)
 	return status, true, nil
 }
 
-func toStatus(lb *elb.LoadBalancerDescription) *api.LoadBalancerStatus {
-	status := &api.LoadBalancerStatus{}
+func (s *AWSCloud) toStatus(lb *elb.LoadBalancerDescription, apiService *api.Service) *api.LoadBalancerStatus {
+	var ingress []api.LoadBalancerIngress
 
 	if !isNilOrEmpty(lb.DNSName) {
-		var ingress api.LoadBalancerIngress
-		ingress.Hostname = orEmpty(lb.DNSName)
-		status.Ingress = []api.LoadBalancerIngress{ingress}
+		var dnsIngress api.LoadBalancerIngress
+		dnsIngress.Hostname = orEmpty(lb.DNSName)
+		ingress = append(ingress, dnsIngress)
 	}
 
-	return status
+	cname := s.getLoadBalancerCname(apiService)
+	glog.Info("CName for service %v is %v", apiService.Name, cname)
```

Comment:
Info is pretty high priority for a purely informative message.  I suggest `glog.V(4).Infof`

Also note that it must be Infof (not Info) to format (I'm always making this mistake!)

Also, I like to use %q, which is like %s but wraps the string in quotes - here because cname might be the empty string it would be much clearer to use %q.  Though I wonder if we should actually only log the cname if it is non-empty, because `CNAME for service hello is ""` would be confusing.  It's fine also to add another log statement if there is no cname.